src/ol_openedx_rapid_response/tools.py

```diff
@@ -0,0 +1,13 @@
+def get_display_name_from_usage_key(key, course):
```

Comment:
Nit: It's common practice for us to use `utils.py` as the name for files like this. If there's a good reason to call it `tools.py`, please ignore

Reply:
This name was being used already before the plugin migration so I kept using it but I agree with you and I've renamed it to `utils.py`.